lib/workers/repository/onboarding/branch/index.ts

```diff
@@ -36,6 +40,21 @@ export async function checkOnboardingBranch(
   setGitAuthor(config.gitAuthor);
   const onboardingPr = await getOnboardingPr(config);
   if (onboardingPr) {
+    isConflicted = await scm.isBranchConflicted(
+      config.baseBranch!,
+      config.onboardingBranch!
+    );
+    if (isConflicted) {
```

Comment:
Shouldn't this check happen after the onboarding rebase check?

Reply:
If branch is conflicted ew don't rebase so better to check it before